os_net_config/utils.py

```diff
@@ -649,6 +650,21 @@ def get_vf_devname(pf_name, vfid):
     return vf_nic[0]
 
 
+def backup_map_files(backup_path):
+    src_files = [common.DPDK_MAPPING_FILE,
```

Comment:
Maybe add a debug message saying "Backing up DPDK, SR-IOV, and mapping files." here.